templates/locales/en.yml

```diff
@@ -877,6 +877,14 @@ en:
         the issues below and execute "vagrant reload":
 
         %{output}
+      nfs_exports_failed: |-
+        Vagrant failed to install an updated NFS exports file. This may be
+        due to overly restrictive permissions on your NFS exports file. Please
+        validate them and try again.
```

Comment:
It seems like the only case where this can happen is if /etc/exports isn't readable by the user. Writing, chowning and moving are optionally prefixed by sudo. If reading optionally used sudo as well, then it would cover the case where /etc/exports is unreadable by normal users.